includes/discovery/sensors/pre-cache/weos.inc.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
Do not use pre-cache.  It is harmful.

Instead use:
`\SnmpQuery::cache()->walk('IF-MIB::ifName')->table();`

The second query is only used in one place and should not be cached.

Reply:
Hi. I'm not sure where to use that instead, but I have tried to rewrite the SNR sensor discovery in YAML instead of PHP, but I'm not able to get it to work. I wrote a response above in PipoCanaja's comment. Thanks.